src/time/format/formatter.cr

```diff
@@ -216,5 +220,30 @@ struct Time::Format
       io.write_byte padding.ord.to_u8 if value < 1000
       pad3 value, padding
     end
+
+    def pad5(value, padding)
```

Comment:
What are pad5 trough 8 for? I don't see any usecase for these. It's certainly easier to use log10 for pad9 instead of dereasingly calling these methods.

Reply:
Yes, it look like strange code duplication because numbers aren't used. Will collapse.